python/semantic_kernel/functions/__init__.py

```diff
@@ -0,0 +1,7 @@
+from semantic_kernel.functions.kernel_function import KernelFunction
```

Comment:
missing header (let's figure out how to have a header automatically added to new files, as a separate item).